src/lib/redux/siteData.ts

```diff
@@ -67,4 +69,16 @@ const getPaymentsForCustomer = async (customer: CustomerRecord): Promise<Payment
   return payments;
 };
 
+// Helper function to grab a tariff plan for a specific customer
+const getTariffPlanForCustomer = async (customer: CustomerRecord): Promise<TariffPlanRecord[]> => {
+  const tariffIds = customer.tariffPlansId;
+  let tariffPlans: TariffPlanRecord[] = [];
+
+  if (tariffIds) {
+    tariffPlans = await getTariffPlansByIds(tariffIds);
+  }
+  
+  return tariffPlans;
+};
+
```

Comment:
Data fetching like for tariff plans were moved to https://github.com/calblueprint/meepanyar-node/blob/main/resolvers/index.js